cmd/client-keystone-auth/main.go

```diff
@@ -84,40 +84,40 @@ func prompt(url string, domain string, user string, password string) (gopherclou
 		if err != nil {
 			return options, err
 		}
-	} else {
-		fmt.Fprintf(os.Stderr, "Keystone auth url is \"%s\"\n", url)
```

Comment:
According to the reported issue, the above message in the console seems a little odd, so it is nice to remove the line in this PR,